integreat_cms/cms/templates/pois/poi_form_sidebar/barrier_free_box.html

```diff
@@ -0,0 +1,18 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load content_filters %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    smile
```

Comment:
```suggestion
    accessibility
```
Maybe this icon would fit even better